.pre-commit-config.yaml

```
@@ -5,7 +5,7 @@ repos:
   rev: v4.5.0
   hooks:
     - id: check-added-large-files
-      args: [ '--maxkb=512', '--enforce-all' ]
+      args: [ '--maxkb=768', '--enforce-all' ]
```

If you increased this size due to the increased size of `conda-lock.yml`, then I suggest configuring the hook as follows, instead. This allows `conda-lock.yml` to grow unbounded, but keep the file size bounded for all other files:

```suggestion
      args: [ '--maxkb=512', '--enforce-all' ]
      exclude: conda-lock.yml
```

---

Ah nice, I was looking to see if there was an option like that under https://github.com/pre-commit/pre-commit-hooks/tree/v4.5.0?tab=readme-ov-file#check-added-large-files, but should have looked at https://pre-commit.com/#config-exclude instead!